addons/point_of_sale/static/src/backend/pos_kanban_view/pos_kanban_view.xml

```diff
@@ -10,7 +10,7 @@
         <div class="d-flex flex-column">
             <t t-call="web.KanbanRenderer" />
             <div class="position-relative container" t-att-class="{ 'border-top mt-2': showTopBorder() }" t-if="this.posState.show_predefined_scenarios">
-                <div class="flex-grow-1 text-center" t-att-class="{ 'disable-buttons': !posState.has_chart_template || loadScenario.status === 'loading' }">
+                <div t-if="this.props.archInfo.activeActions.delete" class="flex-grow-1 text-center" t-att-class="{ 'disable-buttons': !posState.has_chart_template || loadScenario.status === 'loading' }">
```

Comment:
Hey could you explain the reasoning behind this ? I don't really understand how it is linked to the role of the current user
@aamo-odoo